test/utils/mockutils/mockcond/body.go

```diff
@@ -0,0 +1,46 @@
+package mockcond
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"strings"
+)
+
+// Body returns a check expecting body to match template text.
+func Body(expectedBody string) Check {
```

Comment:
Might be easier to deal with bytes than a string

Reply:
Usually request body is written as a string inside test file.
But I can see that this data may come from file.